lektor/packages.py

```diff
@@ -292,7 +332,10 @@ def update_cache(package_root, remote_packages, local_package_path, refresh=Fals
                     package_root, os.path.join(local_package_path, package[1:])
                 )
             else:
-                download_and_install_package(package_root, package, version)
+                download_and_install_packages(
+                    package_root,
+                    ["%s%s%s" % (package, version and "==" or "", version or "")],
+                )
```

Comment:
(I wonder if there's a reason not to accumulate all of the non-local packages and install them in a single pip invocation, rather than installing them one at a time.)

Reply:
I would say that is something for a different pull request. This one should keep it's focus on fixing one thing first.